src/main/webapp/components/basic-search/basic-search.js

```diff
@@ -271,17 +271,17 @@ export const BasicSearchQueryBuilder = props => {
   }
 
   const AddOptions = () =>
-    addOptionsRef ? (
-      ReactDOM.createPortal(
-        <AddButton addFilter={addFilter} />,
-        props.addOptionsRef
-      )
-    ) : (
-      <React.Fragment>
-        <div style={{ marginLeft: '10px' }} />
-        <AddButton addFilter={addFilter} />
-      </React.Fragment>
-    )
+    addOptionsRef
+      ? ReactDOM.createPortal(
+          <AddButton addFilter={addFilter} />,
+          props.addOptionsRef
+        )
+      : addOptionsRef !== null && (
```

Comment:
Are we differentiating between null and undefined here? If so that's kind of strange to me

Reply:
yeah, I thought so too. Maybe I should just make a new component that's independent of basic search